token/program/src/instruction.rs

```diff
@@ -641,6 +662,31 @@ pub fn initialize_account(
     })
 }
 
+/// Creates a `InitializeAccount2` instruction.
+pub fn initialize_account2(
+    token_program_id: &Pubkey,
+    account_pubkey: &Pubkey,
+    mint_pubkey: &Pubkey,
+    owner_pubkey: &Pubkey,
+) -> Result<Instruction, ProgramError> {
+    let data = TokenInstruction::InitializeAccount2 {
+        owner: *owner_pubkey,
+    }
+    .pack(); // TODO do we need to return result?
```

Comment:
Remove TODO?

Reply:
Ah, right.  Copypasta.  I'll just remove all of them